app/assets/javascripts/location_search.js

```diff
@@ -0,0 +1,73 @@
+window.ST = window.ST || {};
+
+(function(module) {
+
+  module.initializeLocationSearch = function() {
+    var input = document.getElementById('q');
```

Comment:
With these old sites not a huge one. Vanilla JS is faster, and the code is easier to be moved somewhere without jQuery support (and my secret plan is to get rid of jQuery in the future since it's way too bloated library).